core/dbt/include/global_project/macros/materializations/models/incremental/incremental.sql

```diff
@@ -33,12 +34,21 @@
   {% set to_drop = [] %}
 
   {% if existing_relation is none %}
-      {% set build_sql = get_create_table_as_sql(False, target_relation, sql) %}
+    {%- call statement('main', language=language) -%}
+      {{ create_table_as(False, target_relation, compiled_code, language) }}
+    {%- endcall -%}
   {% elif full_refresh_mode %}
-      {% set build_sql = get_create_table_as_sql(False, intermediate_relation, sql) %}
```

Comment:
@jtcohen6 any concern moving from `get_create_table_as_sql` to `create_table_as`?

Reply:
Heh — technically, `create_table_as` is the old/originally name, `get_create_table_as_sql` was meant to be more consistent with the fact that macros which don't actually perform a side effect (`run_query`), but simply return code (SQL) as a string, were named `get_x_sql`. It's a convention we have been _very_ imperfect about following.

We made this change in the leadup to v1.0, based on feedback from @dataders (before he worked with us!): https://github.com/dbt-labs/dbt-core/pull/4154#pullrequestreview-794713281